website/static/images/xtable-svg.svg

```diff
@@ -0,0 +1,70 @@
+<svg width="554" height="150" viewBox="0 0 554 150" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
How was the logo produced? If it was created from scratch the we are fine. If it was imported/found from somewhere else then it may not be OK to use it.

Reply:
Hand-crafted from scratch with Figma and exported as SVG format

Reply:
Cool, then there are no LICENSE problems.

Nevertheless, since this is a new logo for the project it would be best to start a VOTE thread in the dev@xtable list before putting this officially online to ensure that there are no objections from the PMC. Can you please start the respective thread? (If you are not familiar with the process I can do that for you)

Other than that the logo must contain also the TM symbol as per [policy](https://www.apache.org/foundation/marks/pmcs#graphics).

Finally, but not blocking for this PR the logo(s) should be submitted in https://www.apache.org/logos/ once the VOTE passes.

Reply:
Thanks for starting the thread. We can do the submission to the logos URL after merging this PR. If you don't have sufficient permissions there let me know and I will do it for you.

Reply:
Ok thank you for the help! I just added commit with TM in the logo now

Reply:
thank you, I have removed all unused images now